src/ui/css/vars.scss

```diff
@@ -36,8 +36,10 @@ $header-accent-color: $link-color;
 $header-text-not-active-color: transparentize($white, 0.5);
 
 // Category Colors
-$category-colors: #fa5252, #e64980, #be4bdb, #7950f2, #4c6ef5, #228ae6, #15aabf,
-  #12b886, #40c057, #82c91e, #fab005, #fd7e14;
+// There are currently 12 colors and this number must be kept in sync with
+// `src/amo/components/Categories/index.js`.
+$category-colors: $teal-80, $green-70, $red-70, $magenta-70, $blue-70,
+  $orange-70, $teal-90, $yellow-80, $purple-70, $green-90, $ink-70, $purple-90;
```

Comment:
TBH, I think the first set of colors you used just look nicer 🌈  They were more vibrant.

Your comment about how `3:1` is acceptable for [active UI components](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast?utm_source=devtools&utm_medium=a11y-panel-checks-color-contrast) definitely applies since these are navigational buttons. I suggest going back to the first palette and only correcting the ones that had less than a `3:1` ratio: `green-60`, `teal-60`, and `orange-50`.

Reply:
I agree the new palette looks.. sad and dark compared to the rest of the site. I wasn't sure about the active components, I'll update the palette.

Related note: I asked for a a11y review a year or so ago but we never did it. Maybe we should think about it again or at least take it into consideration if we update the UI.